airbyte-scheduler/src/main/java/io/airbyte/scheduler/JobTracker.java

```diff
@@ -204,6 +211,25 @@ public void trackSync(Job job, JobState jobState) {
     return metadata.build();
   }
 
+  private ImmutableMap<String, Object> generateJobAttemptMetadata(long jobId, JobState jobState) throws IOException {
+    final Builder<String, Object> metadata = ImmutableMap.builder();
+    final Job job = jobPersistence.getJob(jobId);
+    if (jobState != JobState.STARTED && job != null) {
+      final List<Attempt> attempts = job.getAttempts();
+      if (attempts != null && !attempts.isEmpty()) {
+        final Attempt lastAttempt = attempts.get(attempts.size() - 1);
+        if (lastAttempt.getOutput() != null && lastAttempt.getOutput().isPresent()) {
+          final JobOutput jobOutput = lastAttempt.getOutput().get();
+          final StandardSyncSummary syncSummary = jobOutput.getSync().getStandardSyncSummary();
+          metadata.put("duration", syncSummary.getEndTime() - syncSummary.getStartTime());
```

Comment:
just want to sanity check that sync summary metrics are in seconds?

Reply:
it's in milliseconds based on what's being saved in the syncSummary:
https://github.com/airbytehq/airbyte/blob/0c603db2966ee35b7ed9b62e5c95947522cab429/airbyte-workers/src/main/java/io/airbyte/workers/DefaultSyncWorker.java#L84

Reply:
if you look at the excel spreadsheet that john put together for the tracking plan he is expecting seconds. either his expectation needs to change to ms or we need to convert here. i have no preference, just want to make sure that the two of you are speaking the same language.

Reply:
Ok will switch to seconds, thanks!